src/picongpu/main.cu

```diff
@@ -65,8 +61,8 @@ int main(int argc, char **argv)
             break;
```

Comment:
offline discussion: line 58: the comment looks wrong (should be 0?) and outdated. Maybe we just add a note that the missing break is voluntarily to set the error code to 0 if everything went well.